src/patternfly/components/Skeleton/skeleton.scss

```diff
@@ -0,0 +1,178 @@
+  --pf-c-skeleton__element--after--LinearGradientAngle: 90deg;
+  --pf-c-skeleton__element--after--LinearGradientColorStop1: transparent;
+  --pf-c-skeleton__element--after--LinearGradientColorStop2: #ededed;
+  --pf-c-skeleton__element--after--LinearGradientColorStop3: transparent;
+  --pf-c-skeleton__element--after--TranslateX: -100%;
+  --pf-c-skeleton__element--after--AnimationName: loading;
+  --pf-c-skeleton__element--after--AnimationDuration: 2s;
+  --pf-c-skeleton__element--after--AnimationIterationCount: infinite;
+  --pf-c-skeleton__element--after--AnimationTimingFunction: linear;
+  --pf-c-skeleton__element--after--AnimationDelay: .5s;
+
+  // Circle
+  --pf-c-skeleton__element--m-circle--BorderRadius: var(--pf-global--BorderRadius--lg);
+
+  // Static width and height
+  --pf-c-skeleton__element--m-width-sm--Width: #{pf-size-prem(100px)};
+  --pf-c-skeleton__element--m-width-md--Width: #{pf-size-prem(200px)};
+  --pf-c-skeleton__element--m-width-lg--Width: #{pf-size-prem(300px)};
+  --pf-c-skeleton__element--m-height-sm--Height: #{pf-size-prem(100px)};
+  --pf-c-skeleton__element--m-height-md--Height: #{pf-size-prem(200px)};
+  --pf-c-skeleton__element--m-height-lg--Height: #{pf-size-prem(300px)};
+
+
+  // Text modifiers
+  --pf-c-skeleton__element--m-text-4xl--Height: calc(var(--pf-global--FontSize--4xl) * var(--pf-global--LineHeight--sm));
+  --pf-c-skeleton__element--m-text-3xl--Height: calc(var(--pf-global--FontSize--3xl) * var(--pf-global--LineHeight--sm));
+  --pf-c-skeleton__element--m-text-2xl--Height: calc(var(--pf-global--FontSize--2xl) * var(--pf-global--LineHeight--sm));
+  --pf-c-skeleton__element--m-text-xl--Height: calc(var(--pf-global--FontSize--xl) * var(--pf-global--LineHeight--sm));
+  --pf-c-skeleton__element--m-text-lg--Height: calc(var(--pf-global--FontSize--lg) * var(--pf-global--LineHeight--md));
+  --pf-c-skeleton__element--m-text-md--Height: calc(var(--pf-global--FontSize--md) * var(--pf-global--LineHeight--md));
+  --pf-c-skeleton__element--m-text-sm--Height: calc(var(--pf-global--FontSize--sm) * var(--pf-global--LineHeight--md));
+
+  // Width
+  --pf-c-skeleton__element--m-width-25--Width: 25%;
+  --pf-c-skeleton__element--m-width-33--Width: calc(100% / 3);
+  --pf-c-skeleton__element--m-width-50--Width: 50%;
+  --pf-c-skeleton__element--m-width-66--Width: calc(100% / 3 * 2);
+  --pf-c-skeleton__element--m-width-75--Width: 75%;
+}
+
+.pf-c-skeleton__element {
+  position: relative;
+  height: var(--pf-c-skeleton__element--Height);
+  overflow: hidden;
+  background-color: var(--pf-c-skeleton__element--BackgroundColor);
+  border-radius: var(--pf-c-skeleton__element--BorderRadius);
+
+  &::before {
+    display: block;
+    padding-bottom: 0;
+    content: "";
+  }
+
+  &::after {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    z-index: var(--pf-c-skeleton__element--after--ZIndex);
+    display: block;
+    width: 100%;
+    height: 100%;
+    content: "";
```

Comment:
this is fine, but we typically do `top: 0; right: 0; bottom: 0; left: 0;`. Since we don't have vars currently or if we used the coordinate properties, it doesn't really matter but wanted to mention it.

Reply:
I removed width and height of 100%